enterprise_access/apps/content_assignments/api.py

```diff
@@ -307,8 +312,10 @@ def _update_and_refresh_assignments(assignment_records, fields_changed):
     LearnerContentAssignment.bulk_update(assignment_records, fields_changed)
 
     # Get a list of refreshed objects that we just updated
-    return LearnerContentAssignment.objects.filter(
-        uuid__in=[record.uuid for record in assignment_records],
+    return list(
+        LearnerContentAssignment.objects.filter(
+            uuid__in=[record.uuid for record in assignment_records],
+        )
```

Comment:
Hm, isn't this also sensitive to MySQL statement length limits?  ((1 megabyte) / (40 bytes)) / 10 = 2500 max assignments.

Reply:
In practice, the modern MySQL client limit is 16MB per request, so without the order-of-magnitude safety net that should result in about (16 megabytes) / (40 bytes) = 400,000 max assignments.

Reply:
Ah good point.  I'm going to leave this as-is for now, since anything over 1,000 assignments in a single allocation request is likely going to require a more "bulk-oriented" UX.  I guess we could've said that about the `core.User()` lookup, too.

Reply:
the core.User lookup was more of a bottleneck since the "safe" batch size we landed on was <1000.